packages/react/src/components/drop/DropPod.tsx

```diff
@@ -213,9 +213,12 @@ const RDropPod: FunctionComponent<IRDropPodProps> = ({
             // Restrict the max-width to the inner width of the closest relatively positioned ancestor
             if (relativeParent) {
                 const {paddingLeft, paddingRight} = getComputedStyle(relativeParent) ?? {};
+                const fallbackMaxWidth = 850;
+                const calculatedMaxWidth = parentOffset.width - (parseFloat(paddingLeft) + parseFloat(paddingRight));
+                const maxWidth = !!isNaN(calculatedMaxWidth) ? fallbackMaxWidth : calculatedMaxWidth;
```

Comment:
isNaN is already a boolean, no need for the `!!`
```suggestion
                const maxWidth = isNaN(calculatedMaxWidth) ? fallbackMaxWidth : calculatedMaxWidth;
```